### ISSUE TITLE
New idea, API messages

### ISSUE THREAD

**Issue by netRunner0** [merdok/homebridge-webos-tv#550]
Thanks again for your work, the plugin is great!

You have support for messages on the TV in the form of static buttons and predefined messages.
It would be nice if you made API support for messages, so that it would be possible to send any messages with any data via HomeKit automation (http requests).

Please consider this as an idea for implementation)

**Comment by merdok** [comment]
thanks for the feedback, but i am not quite sure if i understand it correctly. Could you please elaborate it more, if possible with an real world use case?

**Comment by netRunner0** [comment]
For example, I need to send a message about low water level (25%) or about upcoming frosts (up to -20)

The values ​​25 and 20 are dynamic. They are in Homekit, and I would like to send them to TV via Homekit

**Comment by merdok** [comment]
There is a `file` property on the `notificationButtons`, when you specify that, it will read out the file and show the content of it in the notification. Other then that you can use the `toast` cli command which will also show a notification on the tv.

**Comment by netRunner0** [comment]
This is good, but there is no way to write a file in Homekit automations. It would be nice to have a web server on some port to receive these commands via POST.

**Comment by netRunner0** [comment]
I tried to access from commands via SSH but there was a problem
To get to the CLI, you need to run the command hb-shell and then webostv toast ...

I tried to pass the execution of 2 commands via SSH from commands:
sudo hb-shell & webostv toast ip mac message but this returns an unknown error

When simply working from the console, messages are sent fine

**Comment by netRunner0** [comment]
![IMG_6655](https://github.com/user-attachments/assets/49eb11e8-bf8c-4074-9837-0691122b5f97)

**Comment by merdok** [comment]
This seems to be more of an OS related issue and would have nothing to do with the initial request, so i would recommend you to check the permissions or other OS related things.

**Comment by netRunner0** [comment]
The fact is that one command is always executed, and two commands in a row can no longer be executed (using shorts)

Therefore, I would like to see an intermediary in the form of a mini web server to receive messages via json...

**Comment by merdok** [comment]
Yeah, you would need to build yourself that small web server if you require it, the tools to do that are available. If you think that some other people could benefit from that then you could submit a pull request and we could include that in the repo.

**Comment by jmasarweh** [comment]
Love your plugin and I would really benefit from having an API endpoint to update the Notification Message on the fly. Right now the Homebridge API allows you to only control the state of the TV Notification Buttons (On/OFF) with no access to the "message:" attribute. 
The command line option is very limited and has a lot of friction points (i,e, shell limitation, hombridge witout hub-shell, ios shortcut limitation)....

Thank you!
The JSON body could be as simple as this:
```
body {
"buttonName": string, 
"message": string
}
```

**Comment by merdok** [comment]
thanks for the suggestion! will see that i can do!
